INSTALL.md

```diff
@@ -1,41 +1,66 @@
-# HOWTO install/test Folksonomy API
+# Folksonomy API Installation Guide
+  This guide will walk you through installing and setting up the Folksonomy API on a fresh Debian 12 system.
 
-## Requirements
+ ## Prerequisites
+ A fresh Debian 12 installation
 
-- Postgresql 13 (lower version may be OK, but untested)
-- Python 3.8 (lower version may be ok too, but untested)
-- Python modules in "requirements.txt"
+ Root access or a user with sudo privileges
 
-## Setup
+ Basic knowledge of the command line
+
+## Step 1: Install required packages
+
+```apt install git sudo postgresql python3-venv -y```
+
+
+## Step 2: Create a New User for Folksonomy API
+For security reasons, it's recommended to run the API under a separate user.
+
+### Create a new user for the Folksonomy API
+   ```adduser folksonomy```
+
+### Add the new user to the sudo group to allow elevated permissions when needed
+```usermod -aG sudo folksonomy```
+
+### Switch to the newly created user:
+```su folksonomy```
+
+# Step 3: Clone the Repository
+## Navigate to your home directory and clone the Folksonomy API repository.
 
 ```
-# clone repo
+cd ~
 git clone https://github.com/openfoodfacts/folksonomy_api.git
 cd folksonomy_api
+```
 
-# required packages to setup a virtualenv (optional, but recommended)
-apt install python3-virtualenv virtualenv virtualenvwrapper
-
-# create and switch to virtualenv
-# if mkvirtualenv command is not found, search for virtualenvwrapper.sh 
-# (/usr/share/virtualenvwrapper/virtualenvwrapper.sh, or /usr/bin/virtualenvwrapper.sh, for example)
-# add the path in your bash profile
-mkvirtualenv folksonomy -p /usr/bin/python3
-workon folksonomy
+# Step 4: Set Up a Python Virtual Environment
+### To isolate the API's dependencies, create and activate a Python virtual environment:
 
-# install 
-pip install -r requirements.txt
 ```
-If you install PostgreSQL yourself, here is how to set it up:
-```bash
-# create dbuser if needed
-sudo -u postgres createuser $USER
+python3 -m venv folksonomy
+. ./folksonomy/bin/activate
+```
+
+# Step 5: Install Python Dependencies
+### Once the virtual environment is activated, install the required Python dependencies from requirements.txt.
+
+```pip install -r requirements.txt```
+
+# Step 6: Set Up PostgreSQL Database
```

Comment:
Here I think you must explain the two options:
* either on a local postgre
* either using `./start_postgres.sh`

The documentation should clearly explain both cases.